src/sentry/static/sentry/app/data/forms/accountNotificationSettings.jsx

```diff
@@ -7,10 +7,10 @@ export const fields = {
   subscribeByDefault: {
     name: 'subscribeByDefault',
     type: 'boolean',
-    label: t('Send Me Project Alerts'),
+    label: t('Send Me Alerts'),
     // TODO(billy): Make this a real link
-    help: tct('Alerts are defined in [locationPath]', {
-      locationPath: '[Project] » Project Settings » Alerts » Rules.',
+    help: tct('Alerts are defined in [locationPath]. Disable this to not receive alerts sent to teams you are in. You will continue to receive alerts sent directly to you.', {
```

Comment:
How do we disable alerts sent directly to you then?

Reply:
For the current iteration of alerts we want them to be removed from the alert configuration instead of users being able to suppress things explicitly sent to them. For more "automatic" / implicit targeting like those done for suspect commits (down the line), we'll have to figure it out. Thoughts?

Reply:
@billyvg lmk if you have any further comments

Reply:
(I also changed the text a bit btw)